### ISSUE TITLE
How do I build/run iOS app?

### ISSUE THREAD

**Issue by valeriyo** [chrisbanes/tivi#1980]
This repo structure is different from the latest KMP template (https://kmp.jetbrains.com/) with top-level `composeApp`, `iosApp`, etc. - I'm not sure what I'm looking at yet...

How about a section in README with useful (gradlew) commands -- how to build/run stuff and other tips?

Thanks!

**Comment by chrisbanes** [comment]
Is the top-level `ios-app` not clear?

**Comment by valeriyo** [comment]
It's a "works on my machine" syndrome:

<img width="353" alt="image" src="https://github.com/user-attachments/assets/f48d0349-ec34-4d48-8ff6-c5b33cd93840">

**Comment by valeriyo** [comment]
The latest KMP template also provides iOS target that can be run from Android Studio:

<img width="293" alt="image" src="https://github.com/user-attachments/assets/bb73be17-46d1-4e80-8294-89e2fadc6dae">

which this project doesn't

**Comment by chrisbanes** [comment]
> It's a "works on my machine" syndrome

No, it's the "it's an open source project and I don't owe you any support" syndrome.
